### PR TITLE
global: jsonresolver for dx.doi.org and doi.org hosts

### PR DESCRIPTION
**Pull request by krzysztof**
Signed-off-by: Krzysztof Nowak k.nowak@cern.ch

### CODE REVIEW

**Comment by krzysztof** [comment]
@lnielsen Actually don't merge this yet, I'll try to integrate #29 and then make a release

**Comment by lnielsen** [comment]
Ok....ping me when done.

File: invenio_openaire/loaders.py [hunk 1]
```diff
@@ -221,28 +223,45 @@ class LocalOAIRELoader(BaseOAIRELoader):
...
+
+    XML -> XML
+    XML -> JSON
+    JSON -> JSON
     """
 
     def __init__(self, source=None, **kwargs):
-        """Initialize the loader for local database."""
+        """
+        Initialize the loader for local database.
```
Comment (jirikuncar) [inline invenio_openaire/loaders.py]: Please, start just after `"""Initialize ...`

File: invenio_openaire/loaders.py [hunk 2]
```diff
@@ -261,7 +280,7 @@ def __init__(self, source=None, setspec=None, **kwargs):
         self.setspec = setspec or \
             current_app.config['OPENAIRE_OAIPMH_DEFAULT_SET'],
 
-    def iter_grants(self):
+    def iter_grants(self, as_json=True):
```
Comment (jirikuncar) [inline invenio_openaire/loaders.py:282]: What about creating `iter_raw_grants` instead and changing implementation as follows?

``` python
def iter_grants(self):
    for grant in self.iter_raw_grants():
        yield self.grantxml2json(etree.fromstring(grant))
```

Comment (krzysztof) [reply invenio_openaire/loaders.py:282]: :+1: for the suggestion but at the moment I'd rather not refactor it anymore.

**Pull request merged**